components/Project.js

```diff
@@ -0,0 +1,32 @@
+import styled from "styled-components";
+import { titles } from "../lib/data.json";
+import { v4 as uuidv4 } from "uuid";
+import ToolsIcon from "../public/tools.svg";
+
+export default function Project() {
```

Comment:
'Project' might not be descriptive enough. How about ProjectItem or ProjectPost?

Reply:
you right! changed it :)